contrib/authors_update.sh

```diff
@@ -26,7 +26,8 @@ tmp=$(mktemp --tmpdir=./)
 grep @ AUTHORS >"$tmp"
 
 echo "Names:"
-
+new_name=""
```

Comment:
Unrelated?

Reply:
![image](https://github.com/user-attachments/assets/eff7aab6-8394-4112-b10a-ba327146a9ad)
CI was failing because I wasn't in the AUTHORS file, added myself
See https://github.com/openucx/ucx/pull/10308#issuecomment-2511630243